src/services/chirpstack/network-server.service.ts

```diff
@@ -68,4 +69,9 @@ export class ChirpstackSetupNetworkServerService
 
         return createNetworkServerDto;
     }
+
+    public async getAdrAlgorithmsForDefaultNetworkServer(): Promise<ListAllAdrAlgorithmsResponseDto> {
+        const networkServerId: string = await this.getDefaultNetworkServerId();
```

Comment:
Strictly not necessary to specify type as the method return type is enforced

Reply:
Thank you for the comment. I am not completely sure if it is just the explicit type declaration for the string you

Reply:
That's exactly it.